Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/covidcertificate/CertificatesFragmentTest.kt

```diff
@@ -0,0 +1,190 @@
+package de.rki.coronawarnapp.covidcertificate
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import dagger.Module
+import dagger.android.ContributesAndroidInjector
+import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.covidcertificate.test.ui.CertificatesFragment
+import de.rki.coronawarnapp.covidcertificate.test.ui.CertificatesViewModel
+import de.rki.coronawarnapp.covidcertificate.test.ui.cards.CovidTestCertificateCard
+import de.rki.coronawarnapp.covidcertificate.test.ui.cards.CovidTestCertificateErrorCard
+import de.rki.coronawarnapp.covidcertificate.test.ui.items.CertificatesItem
+import de.rki.coronawarnapp.covidcertificate.vaccination.core.VaccinatedPerson
+import de.rki.coronawarnapp.covidcertificate.vaccination.ui.cards.CreateVaccinationCard
+import de.rki.coronawarnapp.covidcertificate.vaccination.ui.cards.HeaderInfoVaccinationCard
+import de.rki.coronawarnapp.covidcertificate.vaccination.ui.cards.ImmuneVaccinationCard
+import de.rki.coronawarnapp.covidcertificate.vaccination.ui.cards.NoCovidTestCertificatesCard
+import de.rki.coronawarnapp.covidcertificate.vaccination.ui.cards.VaccinationCard
+import io.mockk.MockKAnnotations
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import org.joda.time.DateTime
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import testhelpers.BaseUITest
+import testhelpers.Screenshot
+import testhelpers.launchFragment2
+import testhelpers.launchInMainActivity
+import testhelpers.selectBottomNavTab
+import testhelpers.takeScreenshot
+
+@RunWith(AndroidJUnit4::class)
+class CertificatesFragmentTest : BaseUITest() {
+
+    @MockK lateinit var viewModel: CertificatesViewModel
+    @MockK lateinit var vaccinatedPerson: VaccinatedPerson
+
+    @Before
+    fun setup() {
+        MockKAnnotations.init(this, relaxed = true)
+        every { vaccinatedPerson.fullName } returns "Andrea Schneider"
```

Comment:
Can we use `Max Mustermann` to align with all screenshots?

Reply:
I'm using data according to the Figma design, but I can change the name if we are already using something else somewhere else.